src/app/management/containers/equipment-registration/equipment-registration.component.ts

```diff
@@ -163,4 +184,14 @@ export class EquipmentRegistrationComponent implements OnInit, OnDestroy {
   private getPetSize(): Observable<PetSize[]> {
     return this.controller.getPetSizes();
   }
+
+  private getEquipmentInventoryNumbers() {
+    this.inventoryNumbersSubscription$ = this.controller
+      .getAllEquipment()
+      .pipe(
+        map((res) => res.items.map((item) => item.inventoryNumber)),
+        tap((arr) => (this.inventoryNumbers = arr)),
+      )
+      .subscribe();
```

Comment:
> do not leave subscribe empty. move logic from tap to subscribe.

done


> don't we have an other way to check if the number exist instead of this mislabel one?

unfortunately, i didn't find another way. Let's discuss it during one of the calls, please.